macros/extract_url_parameter.sql

```diff
@@ -0,0 +1,20 @@
+{% macro extract_url_parameter(field, url_parameter) -%}
+
+{{ adapter.dispatch('extract_url_parameter', 'fivetran_utils') (field, url_parameter) }}
+
+{% endmacro %}
+
+
+{% macro default__extract_url_parameter(field, url_parameter) -%}
+
+{{ dbt_utils.get_url_parameter(field, url_parameter) }}
+
+{%- endmacro %}
+
+
+{% macro databricks__extract_url_parameter(field, url_parameter) -%}
```

Comment:
It looks like in all our other macros that include a dispatch for databricks, we provide the `spark` dispatch:
- [json_parse](https://github.com/fivetran/dbt_fivetran_utils/blob/c536047a06153abfed46c272483bd768b2d34263/macros/json_parse.sql#L38)
- [try_cast](https://github.com/fivetran/dbt_fivetran_utils/blob/c536047a06153abfed46c272483bd768b2d34263/macros/try_cast.sql#L50)
- [timestamp_add](https://github.com/fivetran/dbt_fivetran_utils/blob/f7567295c8f406946dfdbcdd761affad7d7bb535/macros/timestamp_add.sql#L41)

Should we be using `spark` dispatch as opposed to `databricks`?

It looks like the `databricks` dispatch inherits the `spark` dispatch (see ref [here](https://docs.getdbt.com/reference/dbt-jinja-functions/dispatch#adapter-inheritance)). With this, I believe we should leverage the `spark` dispatch so this will work regardless of a customer leveraging the spark or databricks adapter. Thoughts?

Reply:
Updated!